number_search.py

```diff
@@ -2,26 +2,46 @@
 #  in article.txt
 # use regex to match numbers and fractions
 import re
+frac_pattern = r'(-?\d+\\[A-Za-z]+{\d+}{\d+} |-?\\[A-Za-z]+{\d+}{\d+})'
```

Comment:
this pattern will capture any text that starts with \\ .... try specifying the text frac or tinyfrac